server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetMappingsResponse.java

```diff
@@ -69,6 +69,10 @@ public GetMappingsResponse(ImmutableOpenMap<String, MappingMetadata> mappings) {
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
+        writeMappingMetadata(out, mappings);
+    }
+
+    public static void writeMappingMetadata(StreamOutput out, ImmutableOpenMap<String, MappingMetadata> mappings) throws IOException {
```

Comment:
Maybe this should exist in MappingMetadata, rather than a random response class?

Reply:
++